### PR TITLE
add Apache Spark

### PR DESCRIPTION
**Pull request by mengxr**
Plan for dropping Python 2 support in Apache Spark was announced today at http://spark.apache.org/news/plan-for-dropping-python-2-support.html. This PR adds Apache Spark to the project list.

cc: @srowen @felixcheung @HyukjinKwon @gatorsmile @mateiz

Screen Shot 2019-06-03 at 11 41 07 AM

Screen Shot 2019-06-03 at 11 41 39 AM

### CODE REVIEW

**Comment by mengxr** [comment]
cc @Carreau

File: _sections/30-projects.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ These projects pledge to drop Python 2 support in or before 2020.
 - [![](assets/tensorflow.png)TensorFlow](https://www.tensorflow.org/) <!-- url:https://github.com/tensorflow/tensorflow sg:128577 -->
 - [![](assets/requests.png)Requests](https://github.com/kennethreitz/requests) <!-- url:https://github.com/kennethreitz/requests sg:38793 -->
 - [![](assets/scikit-learn.png)scikit-learn](https://scikit-learn.org/) <!-- url:https://github.com/scikit-learn/scikit-learn sg:35400 -->
+- [![](assets/apache_spark.png)Apache Spark](https://spark.apache.org/) <!-- url:https://github.com/apache/spark sg:22104 -->
```
Comment (felixcheung) [inline _sections/30-projects.md]: shouldn't this be added to the end of the current list?

Comment (mengxr) [reply _sections/30-projects.md]: It is ordered by github stars.

**Comment by takluyver** [comment]
Thanks! Can you run `scripts/squash-images.sh` on the logo? This will resize and compress it to save bandwidth.

**Comment by Carreau** [comment]
> Thanks! Can you run `scripts/squash-images.sh` on the logo? This will resize and compress it to save bandwidth.

Done; it did not improve much though.

**Comment by Carreau** [comment]
> Plan for dropping Python 2 support in Apache Spark was announced today 

Congratulation ! I'm squashed the image and all seem good !

**Pull request merged**